generator-scrivito/generators/obj/templates/XEditingConfig.js

```diff
@@ -0,0 +1,20 @@
+import * as Scrivito from "scrivito";
+import {
+  metaDataEditingConfigAttributes,
+  metaDataInitialContent,
+  metaDataPropertiesGroup,
+} from "../_metaDataEditingConfig";
+
+Scrivito.provideEditingConfig("<%= nameUpper %>", {
+  title: "<%= nameUpper %>",
+  attributes: {
+    title: {
+      title: "Name",
+    },
+  },
+  properties: ["title", "description", "image"],
```

Comment:
Please extend attributes with `metaDataEditingConfigAttributes`, improve the title description and remove "description" and "image" from the properties:

```js
  attributes: {
    ...metaDataEditingConfigAttributes,
    title: {
      title: "Title",
      description: "Limit to 55 characters.",
    },
  },
  properties: ["title"],
```